ussp-api/utm-adapter.yaml

```diff
@@ -289,6 +289,123 @@ paths:
       tags:
         - flight-authorization
 
+  /qgcs/geo_zone:
+    get:
+      summary: 'Get a list of all available GeoZones'
+      security:
+        - AuthFromOAUTH2: [qgcs.utm_adapter.read]
```

Comment:
Does oauth2.0 scope for Mission planner client (qgcs) vs  opensource visualization client(spotlight) need to be mentioned explicitly. Is qgcs expected to air traffic  visualization capability using the below interfaces.

Reply:
Thank you for the comment. The idea is yes air traffic should be visible in QGCS. The other scopes are not relevant here e.g. Spotlight so at the moment there is no need we can revisit this again later as well.